src/SFA.DAS.EAS.Web/Controllers/EmployerCommitmentsController.cs

```diff
@@ -419,22 +419,22 @@ public async Task<ActionResult> CreateCommitment(CreateCommitmentViewModel viewM
         [Route("{hashedCommitmentId}/details")]
         public async Task<ActionResult> Details(string hashedAccountId, string hashedCommitmentId)
         {
-
             if (!await IsUserRoleAuthorized(hashedAccountId, Role.Owner, Role.Transactor))
                 return View("AccessDenied");
 
-            var model = await Orchestrator.GetCommitmentDetails(hashedAccountId, hashedCommitmentId, OwinWrapper.GetClaimValue(@"sub"));
-
-            if (!model.Data.IsReadOnly && _featureToggleService.Get<EnhancedApprovals>().FeatureEnabled)
+            if (_featureToggleService.Get<EnhancedApprovals>().FeatureEnabled)
```

Comment:
I am surprised there were no unit tests broken by this (not _hugely_, mind). Can you verify that there are no tests? If not, don't worry about it, as this endpoint will soon be redundant.

Reply:
Yes, I looked at the unit test, but didn't find any written to cover it.